package.json

```diff
@@ -27,11 +27,11 @@
 		"startd": "nodemon src/",
 		"debug": "nodemon --inspect=5959 src/",
 		"debug:container": "nodemon --inspect=0.0.0.0:5959 src/",
-		"coverage": "cross-env NODE_ENV=test nyc --reporter=text-summary mocha --exit './test/**/*.test.js'",
+		"coverage": "cross-env NODE_ENV=test nyc --reporter=text-summary mocha --exit './{,!(node_modules)/**/}*.test.js'",
```

Comment:
These changes should not be required at all. Why did you blacklist node_modules here explicitly? There could be added a single mocha configuration file instead (this will appear when switching to typescript)

Reply:
Why not saying where to find tests instead of where not to find them? `./(test|src)/**/*.test.ext` @Metauriel

Reply:
I tried that before, and it didnt work well... ;)